### ISSUE TITLE
Volunteers Wanted - Overview of the Engine Team

### ISSUE THREAD

**Issue by CharliePoole** [nunit/nunit-console#1221]
As you can see below, the __Engine Team__ has a lot of work to do! We're a small team of volunteers and we welcome __new members__ who want to help on any of our projects. If you are just starting, we'll guide you as you learn about our projects. If you have extensive experience, we can probably find a few challenges for you. Contact @CharliePoole if you would like to join us.

### What We Support

The __NUnit Engine__ is the main component we support. The engine is used by almost all NUnit runners to load and run tests. It's a crucial bit of software, although somewhat invisible to the average user by design. The engine basically decides how and where to run your NUnit tests. 

Along with the engine, we maintain the __NUnit3 Console Runner__. This was once the primary runner used to execute NUnit tests. Nowadays, most developers run their tests on the desktop in other ways but the console runner still has a significant place in the CI environment. In addition, it plays a valuable role in testing the engine itself and its code serves as a benchmark implementation for the development of any NUnit runner.

The engine team also maintains a number of separately built and installed __Engine Extensions__:
  * The __NUnit Project Loader__, which allows use of projects of type `.nunit`.
  * The __VS Project Loader__, which allows use of VS projects and solutions.
  * The __NUnit V2 Result Writer__, which creates output using the NUnit V2 format.
  * The __NUnit V2 Framework Driver__, which allows loading and running NUnit V2 tests.

#### NuGet

* NUnit.ConsoleRunner
* NUnit.ConsoleRunner.NetCore
* NUnit.Console
* NUnit.Engine
* NUnit.Engine.Api
* NUnit.Extension.NUnitProjectLoader
* NUnit.Extension.VSProjectLoader
* NUnit.Extensino.NUnitV2ResultWriter
* NUnit.Extension.NUnitV2Driver

#### Chocolatey.Framework.
* nunit-console-runner
* nunit-extension-nunit-project-loader
* nunit-extension-vs-project-loader
* nunit-extension-nunit-v2-result-writer
* nunit-extension-nunit-v2-driver

#### MSI
* NUnit.Console

#### Zip
* NUnit.Console.Zip

**Comment by stevenaw** [comment]
I'm a little late replying, but have kept this in my inbox until I could.
I've been intending and wanting to get more involved in the engine for a while now, however the main thing keeping me from it has been a lack of spare time. Hopefully over the coming months things will free up a bit and I'll be able to contribute. 🙂

**Comment by CharliePoole** [comment]
If you're able, that would be great!

**Comment by manikumarnune123** [comment]
I'm very late, but I am willing to learn and contribute.

**Comment by CharliePoole** [comment]
@manikumarnune123 So am I, I'm afraid! Still interested?

@stevenaw If things have freed up, take a look at the 3.19 milestone and let me know if anything is of interest.

**Comment by stevenaw** [comment]
Thank you for following up @CharliePoole 
Unfortunately, no, things have not freed up as much as I'd need them to. However, I'd still like to take the opportunity now to thank you for your effort in getting the 3.18.x console runner releases sorted this past while.

**Comment by stevenaw** [comment]
I will continue to keep an eye on things, of course, and will pick up an item if time permits. 😃

**Comment by CharliePoole** [comment]
Fair enough. If you can find any time now, cast an eye on the 3.19 milestone. I decided to be a bit more structured and spell out the content in advance, with priorities. I'd appreciate any general comments (what's missing for example) as well as internal comments, esp. on those things labeled as needing discussion.

I appreciate the thanks. :-)
